sos/plugins/rpm.py

```diff
@@ -32,6 +32,7 @@ class Rpm(Plugin, RedHatPlugin):
         'java.*', 'perl.*',
         'rpm', 'yum',
         'spacewalk.*',
+        'zlib', 'openssl.*', 'nss.*',
```

Comment:
Ack to the packages, but nack to the patch :-D

I kinda overtook this work this week with the new per-plugin verify changes. I think it makes more sense to rebase this on top of that and merge it to master with the rest of those changes.

I'll get a patch added in my tree.